cranelift/codegen/meta/src/shared/instructions.rs

```diff
@@ -1041,6 +1041,123 @@ pub(crate) fn define(
         .can_store(true),
     );
 
+    let I16x8 = &TypeVar::new(
+        "I16x8",
+        "A SIMD vector with exactly 8 lanes of 16-bit values",
+        TypeSetBuilder::new()
+            .ints(16..16)
+            .simd_lanes(8..8)
+            .includes_scalars(false)
+            .build(),
+    );
+    let a = &Operand::new("a", I16x8).with_doc("Value loaded");
+
+    ig.push(
+        Inst::new(
+            "uload8x8",
```

Comment:
Maybe overload the existing `uload8` as `uload8.i16x8`?

Reply:
The controlling type is already occupied by the address type (`i32` or `i64`); e.g. `uload8x8 v1` actually becomes `uload8x8.i64` if `v1` is an `i64` address.